service/src/test/java/bio/terra/pipelines/BaseSpringBootTest.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.pipelines;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest
+@ActiveProfiles({"test", "human-readable-logging"})
+// @ContextConfiguration(classes = App.class)
```

Comment:
I'm not sure if I commented this out or if you did. Just wanted to point it out in case it was me. I'm not sure what it exactly does.

Reply:
ah right, I did - thanks, I'll remove it